addons/l10n_in/models/account_invoice.py

```diff
@@ -188,7 +188,7 @@ def _l10n_in_get_hsn_summary_table(self):
             base_lines.append({
                 'l10n_in_hsn_code': line.l10n_in_hsn_code,
                 'quantity': line.quantity,
-                'price_unit': line.price_unit,
+                'price_unit': line.currency_id.round(line.price_unit * (1 - (line.discount / 100.0))),
```

Comment:
You must not do something on one language that is not in the other -_- if you round python-side, you need to round js-side too. I don't think you want to loose some extra precision here.

Reply:
I have added rounding in js-side too

Reply:
@niai-odoo But are you sure about your rounding here? If the user decided to express the price unit using a decimal precision of 5 digits, I think you have to use 5 digits here, not 2